s3/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+version: '3.1'
+
+services:
+  minio1:
```

Comment:
Did you implement integration tests against these minio servers?
I so, please send a separate PR with those and remove this section for now.

Reply:
Yes I ran against integration tests these minio servers. I noticed that you have an issue, #938, open to run integration tests in CI, in that case there should be a way to create buckets on startup and delete once done.

I'll remove and will send new PR for this.